app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,86 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?
+      erb :'tweets/tweets'
+    else
+      redirect '/login'
+    end
+  end
+
+    get '/tweets/new' do
+    if logged_in?
+      erb :'tweets/create_tweet'
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweets' do
+    if logged_in?
+      if params["content"].empty?
+        redirect '/tweets/new'
+      end
+      @tweet=Tweet.new(content: params["content"])
+      @tweet.user_id=session[:id]
+      @tweet.save
+      erb :'tweets/show_tweet'
+    else
+      redirect "/login"
+    end
+  end
+
+  get '/tweets/:id' do
+
+    if logged_in?
+      @tweet=Tweet.find(params[:id])
+      # @user=User.find(@tweet.user_id)
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` before allowing them to edit? You could use something like
```
@tweet = current_user.tweets.find_by(id: params[:id])
```